parasol/solr/tests.py

```diff
@@ -60,6 +60,22 @@ def clean_up():
     request.addfinalizer(clean_up)
     return client
 
+@pytest.fixture
+def core_test_client(request):
+    client = SolrClient(**TEST_SETTINGS)
```

Comment:
Test Solr connection settings shouldn't be hardcoded in the test file. Can these go in the `testsettings` I see mentioned in the travis config?